lighthouse-cli/bin.ts

```diff
@@ -187,16 +168,16 @@ if (cliFlags.output === Printer.OutputMode[Printer.OutputMode.json] && !cliFlags
  */
 function initPort(flags: {port: number}): Promise<undefined> {
   return Promise.resolve().then(() => {
-    if (flags.port !== 0) {
-      log.verbose('Lighthouse CLI', `Using supplied port ${flags.port}`);
-      return;
-    }
+  if (flags.port !== 0) {
```

Comment:
indent-wise: this is a bit odd:

![image](https://cloud.githubusercontent.com/assets/39191/25787139/acc344ea-3353-11e7-8bda-6e3756df9daf.png)